### DISCUSSION TITLE
Where can you find recent 8.x cloudserver Docker images?

### DISCUSSION THREAD

**Discussion by bergerst** [scality/Zenko#1780] [Q&A]
On dockerhub, [latest-8.2](https://hub.docker.com/layers/zenko/cloudserver/latest-8.2/images/sha256-c59ce92a459c3a3c405f32312411ff0c27b26d66488f47d6efd124e7ecd64e27?context=explore) was last pushed 9 months ago. Since then, no other 8.x version was tagged.

On the other hand, [8.7.4 was released 5 days ago](https://github.com/scality/cloudserver/releases/tag/8.7.4).

So where can I find 8.7.x images?

Also, what's the difference between 7.10, 7.70 and 8.7?

**Comment by sytolk** [comment]
I think this is the [latest](https://hub.docker.com/layers/zenko/cloudserver/latest-0.0.0/images/sha256-191ff391274c69121d190669dd31a4ffc4aab82aca89ada99a28a735363258fc?context=explore) `image: zenko/cloudserver:latest-0.0.0`

**Comment by bergerst** [comment]
@jonathan-gramain You seem to be the most active cloudserver maintainer. Could you answer my question?

**Comment by sytolk** [comment]


**Reply by bergerst** [reply]
@sytolk Wrong thread? I don't think this has anything to do with my question.

**Comment by sytolk** [comment]
@bergerst its have Dockerfile in 8.7 branch: https://github.com/scality/cloudserver/blob/development/8.7/Dockerfile
you can build v8.7 docker image. Why zenko/cloudserver version is so important for you?

**Comment by mlacko64** [comment]
@sytolk
Hello,
sorry to using your discussion thread, but I have a question, I believe it is related.

I see in docker.io zenko cloudserver with tag "latest", which is but 9 months old. Then there is recently updated "latest-0.0.0" which you also mentioned. So, my question is , should we use "latest-0.0.0", when we want to use latest **stable** build? Or just "latest" ?

I am asking because "latest" is 9 months old and our container security scanning software is reporting vulnerabilities there. Still, usually latest stable containers are tagged just "latest".

So, what is prefered way here, then? Use "latest", "latest-0.0.0" or build container by ourselfves and do not rely on containers available on docker.io ?

Thank you very much for your answer

**Reply by sytolk** [reply]
@mlacko it appears that docker images of the cloudserver is not maintained from the 9 months. There is latest-0.0.0 image -> from the file size it appears to be v7 and latest v8 is 9 months old.. The preffered way is to build you own cloudserver from v8.7 branch.

**Reply by mlacko64** [reply]
Hello,
understand now. Thanks for your reply

**Reply by bergerst** [reply]
@mlacko64 sytolk is not a dev in this project, so don't rely on his statements too much.

I'm still waiting for an actual developer from scality to answer my question.

What I found out in the meantime is that the current build pipeline uploads tags named after the commit hash to the Github Docker registry which are deleted after 2 weeks or so.

ghcr.io/scality/cloudserver/cloudserver:bb62ed4fa7ff092fa3e379a9848f7cec6f55460a is the image for Release 8.7.7.

But since the image will be deleted, it's not recommended to use it.

**Reply by sytolk** [reply]
yes I'm not a dev in this project but can't wait 9 months for the dev from scality to answer the question.. Why not simple build docker image from 8.7 branch? something like: 
`docker build https://github.com/scality/cloudserver.git\#development/8.7`
